DSharpPlus/Net/Rest/DiscordApiClient.cs

```diff
@@ -2094,7 +2094,17 @@ DiscordMessageBuilder builder
                 Files = builder.Files
             };
 
-            RestResponse res = await this._rest.ExecuteRequestAsync(request);
+            RestResponse res;
```

Comment:
```suggestion
            RestResponse res;
            try
            {
                res = await this._rest.ExecuteRequestAsync(request);
            }
            finally
            {
                builder.ResetFileStreamPositions();
            }

            return this.PrepareMessage(JObject.Parse(res.Response!));
```

Reply:
this should also fix the double reset call that seems to be still causing issues